tests/unit/scripting.tcl

```diff
@@ -568,13 +568,19 @@ start_server {tags {"scripting"}} {
     # script command is only relevant for is_eval Lua
     test {SCRIPTING FLUSH - is able to clear the scripts cache?} {
         r set mykey myval
+
+        r script load {return redis.call('get',KEYS[1])}
         set v [r evalsha fd758d1589d044dd850a6f05d52f2eefd27f033f 1 mykey]
         assert_equal $v myval
-        set e ""
         r script flush
-        catch {r evalsha fd758d1589d044dd850a6f05d52f2eefd27f033f 1 mykey} e
-        set e
-    } {NOSCRIPT*}
+        assert_error {NOSCRIPT*} {r evalsha fd758d1589d044dd850a6f05d52f2eefd27f033f 1 mykey}
+
+        run_script {return redis.call('get',KEYS[1])} 1 mykey
```

Comment:
let's use EVAL directly? isn't `run_script` created to share thing with functions (not really meant for this specific test)?